### DISCUSSION TITLE
Bun could be a great improvement of tooling

### DISCUSSION THREAD

**Discussion by chrisbbreuer** [stacksjs/stacks#211] [General]
Hi everyone,

_TLDR: Bun is an "incredibly fast JavaScript runtime, bundler, transpiler and package manager – all in one."_

Upon viewing @Jarred-Sumner's [Vite talk](https://twitter.com/jarredsumner/status/1581427214691708929/photo/1), it sure looks like this could be a great direction to be explored. I really enjoy the direction Bun is headed with their tooling. 

If implemented & deemed stable enough, it would provide the following benefits:

- [faster than Node](https://twitter.com/jarredsumner/status/1581427237877784576/photo/1)
- [faster than pnpm](https://twitter.com/jarredsumner/status/1581427223357071361/photo/1)
- much faster bundler _([14x faster](https://twitter.com/jarredsumner/status/1581427231703719937/photo/1) than esbuild which Vite relies on)_
- native Vite support 
  - [Bun launches dev servers more quickly](https://twitter.com/jarredsumner/status/1581427226217615360/photo/1) 
  - [native HMR support](https://twitter.com/jarredsumner/status/1581427234421628928/photo/1)

Could possibly be added as a v1 milestone item based on how it will develop within the next few weeks 😊

**Comment by chrisbbreuer** [comment]
wip https://github.com/stacksjs/stacks/pull/604
